draft-linker-adem-wg-adem-core.md

```diff
@@ -371,6 +391,8 @@ If this is the case, such endorsements MUST be served alongside the root keys at
 All of a PP's keys, endorsed by third parties MUST be served under their OI.
 PPs MAY use their root keys to sign further, internal endorsements, i.e., endorse keys of their own to either issue emblems or further endorsements.
 
+TODO(djackson) - Is it better to define a HTTPS method of distribution? Why special case the root keys? 
```

Comment:
I forgot to remove this. Prior to including serving the keys via a subdomain, they were served within HTTPS headers.